CHANGELOG.md

```diff
@@ -453,11 +454,17 @@ The `richEditing` mixin can be replaced by just using the `NcRichText` component
 * fix(NcUserStatusIcon): adjust icon colors [\#7392](https://github.com/nextcloud-libraries/nextcloud-vue/pull/7392) \([Antreesy](https://github.com/Antreesy)\)
 * fix(NcUserStatusIcon): adjust scoped names for style variables [\#7396](https://github.com/nextcloud-libraries/nextcloud-vue/pull/7396) \([Antreesy](https://github.com/Antreesy)\)
 
+## [v8.31.0](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.31.0) (unreleased)
+### 📝 Notes
+The `NcSettingsInputText` component was deprecated,
+it is no longer used by Nextcloud or Nextcloud apps and not recommended to be used for design reasons.
+
```

Comment:
Seem not relevant for a `main` branch, as v8 changelog will be changed on `stable8`.

Reply:
I'm copying it back for 9rc releases, so fine